Header header
string part_id  
bool defect_detected
float32 overall_confidence
int32 total_defects
DefectInfo[] defect_details
float32 processing_time_ms 